frontend/www/js/omegaup/components/problem/FinderWizard.vue

```diff
@@ -139,12 +139,14 @@ export default class ProblemFinderWizard extends Vue {
   get tagObjects(): TagObject[] {
     const tagObjects: TagObject[] = [];
     this.possibleTags.forEach((tagObject) => {
-      tagObjects.push({
-        key: tagObject.name,
-        value: Object.prototype.hasOwnProperty.call(T, tagObject.name)
-          ? T[tagObject.name]
-          : tagObject.name,
-      });
+      if (T[tagObject.name]) {
```

Comment:
```suggestion
      if (!Object.prototype.hasOwnProperty.call(T, tagObject.name)) {
        return;
      }
```

idealmente siempre hay que intentar usar el [guard clause pattern](https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html) y mantener el patrón preferido de usar `hasOwnProperty` en vez de intentar acceder directamente la propiedad para ver si está establecida.